frontend/src/metabase/components/ErrorPages/ErrorDiagnosticModal.tsx

```diff
@@ -70,6 +83,54 @@ export const ErrorDiagnosticModal = ({
     onClose();
   };
 
+  const handleSlackSubmit = async (values: PayloadSelection) => {
+    setIsSlackSending(true);
+    const selectedKeys = Object.keys(values).filter(
+      key => values[key as keyof PayloadSelection],
+    );
+    const selectedInfo: ErrorPayload = _.pick(errorInfo, ...selectedKeys);
+
+    try {
+      const response = await fetch("/api/slack/bug-report", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ diagnosticInfo: selectedInfo }),
+      });
```

Comment:
we should define this new endpoint as an RTK query api endpoint in `frontend/src/api`